docs/post-call-survey.md

```diff
@@ -2,6 +2,69 @@
 
 You can configure post-call surveys to collect valuable feedback like quality of services or net promoter score after a call ends.
 
+## Create a 1-question poll
```

Comment:
Last one: this would be a nice addition, right before this section:

```
Post-call survey types:
- [1-Question Poll](#onequestionpoll)
- [MS Forms](#msforms)
- [Custom](#custom)
```

Then we can set the headers as links: https://www.daveabrock.com/2018/03/04/using-anchor-links-in-markdown/

That way you can click on the link and it will take you to the correct section